### ISSUE TITLE
A11y_Visual Studio Code Services_Docs_Left Right Navigation_Reflow: After applying reflow resolution(#320x, 256x), right navigation 'Getting started' links are not visible on the page.

### ISSUE THREAD

**Issue by SachinChavhan317** [microsoft/vscode-docs#6210]
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### GitHubTags:
#WCAG1.4.10; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_Apr2023; #Reflow;

### Environment Details:
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Chrome Version 111.0.5563.147 (Official Build) (64-bit)

### Pre-requisite:

1. Open the web page / screen of the web application that is to be tested.
2. Open Chrome/Edge dev Inspect tools. Keyboard shortcut "Ctrl+Shift+I".
3. Ensure devtools are customized to Dock location: "undock into separate window". Minimize devtools so they are not obscuring the page.
4. Zoom the browser window of the page to be tested to 200%.
5. Adjust the browser window's height and width to the required viewport of 320px*256px. As long as dev tools remain open while resizing the viewport size will be displayed in the upper right corner of the browser so the size can be verified.

### Repro Steps:

1. Hit the application URL: https://code.visualstudio.com/
2. Navigate to "Docs" link.
3. Apply reflow resolution.
4. Left and right navigation will appear.
5. Verify whether the after zooming the page up to 200%, right navigation 'Getting started' links are visible or not.

### Actual Result:
After applying reflow resolution (#20x, 256x), right navigation 'Getting started' links are not visible on the page.

### Expected Result:
After applying reflow resolution (#20x, 256x), right navigation 'Getting started' links should be available on the page.

### User Impact:
Low Vision users will not get this right navigation feature links if getting started links are not available in reflow resolution.

### Attachment:
![Getting started](https://user-images.githubusercontent.com/86241494/230598478-a62f537c-c9bc-4498-90f4-7303f604dd79.gif)

**Comment by rzhao271** [comment]
Hi @SachinChavhan317, I believe this issue is a duplicate of #6205 and can be closed.

In both cases, you are causing the website to adjust into a view meant for smaller screens such as tablets or phones. Therefore, the problem is the same for both issues, which is that we currently hide the right nav bar for those views.

**Comment by rzhao271** [comment]
@SachinChavhan317 The issue should be fixed now. There is now a selectbox at the top of the screen for mobile users to navigate to a section within the page.

**Comment by SachinChavhan317** [comment]
I have verified the bug on URL: https://code.visualstudio.com/docs
I'm not getting any selectbox at the top of the screen for right navigation.
Please refer below attachment for reference.
![6210_Not_Fixed](https://github.com/microsoft/vscode-docs/assets/86241494/bf2bfa49-2796-4f8b-a822-e6e7b11f3859)

**Comment by rzhao271** [comment]
I forgot to modify a set of templates, so the overview page still doesn't have the fix. I'll have it fixed within two days.

**Comment by rzhao271** [comment]
Hi @SachinChavhan317, the issue should be fixed now.

**Comment by rzhao271** [comment]
Closing due to upcoming SLA deadline.

**Comment by SachinChavhan317** [comment]
Verified the bug on URL: https://code.visualstudio.com/docs
Getting started dropdown is available for 'Getting started' controls. But below controls still not available in a reflow resolution.

Adding this screenshot for reference.
![image](https://github.com/microsoft/vscode-docs/assets/86241494/0a278ef5-d230-49fb-a449-ff35851fcf9d)

**Comment by rzhao271** [comment]
Good point. Reopening and will file an extension to incorporate that.

**Comment by rzhao271** [comment]
Hi @SachinChavhan317, I have added some updates to the site that have to do with your screenshot in https://github.com/microsoft/vscode-docs/issues/6210#issuecomment-1576519542. Please take a look again and let me know whether this bug can be closed.

1. We have removed the "Tweet this Link" link from all pages.
2. After reflow, the links are at the bottom of the page. The selectboxes at the top are only for navigating within a section of the website or to another part of the page.

**Comment by SachinChavhan317** [comment]
Verified the bug on URL: https://code.visualstudio.com/docs
The issue is fixed, hence closing the bug.
Adding fixed attachment for reference.
#Closed;
![Presentation1](https://github.com/microsoft/vscode-docs/assets/86241494/368a42c4-9094-4143-be0d-34a0e8cb5d1f)
